src/main/kotlin/Interfaces.kt

```diff
@@ -19,14 +19,15 @@ interface Drivable {
 interface TimeTravelable {
     var charged: Boolean
     fun chargeFluxCapacitor()
+    fun inspectFluxCapacitor()
     fun timeTravel()
 }
 
 abstract class ServiceStation(
     val priceOfGasPerLitre: Double = 2.3
-    ) {
+) {
```

Comment:
There are a number of small formatting changes achieved through the Code > Reformat Code menu item. Always use this to clean up formatting before submission.

Reply:
Will do